### ISSUE TITLE
Large empty space at top of Conversations window?

### ISSUE THREAD

**Issue by clee** [avaidyam/Parrot#100]
Not sure if this is a bug exactly, but it's strangely large to me.

<img width="344" alt="screen shot 2018-03-27 at 3 14 25 pm" src="https://user-images.githubusercontent.com/2089/37995468-d2090776-31d1-11e8-9b4b-ab8d8312dd43.png">

**Comment by avaidyam** [comment]
Haha oops. There's supposed to be a big ol' **Conversations** title there. For now, have you tried going to preferences and merging the conversations list + conversation windows? That should turn off the big empty space.

**Comment by clee** [comment]
Nah, I like having the spatial separated conversation/list windows. (A big part of the reason I was interested in Parrot in the first place was because I miss iChat's UI, and I hate Messages)

**Comment by avaidyam** [comment]
@clee That's fine! Over several iterations of the UI, it's gone back and forth and is now finally modular, so you can take your pick. I'll keep this issue open for now and fix it later - likely for `v1.0-beta`.

**Comment by clee** [comment]
👍
